### PR TITLE
fixed weird ui bug that tore up ui on hiding of load handler

### PR DESCRIPTION
**Pull request by temitope**

### CODE REVIEW

**Comment by temitope** [comment]
probably unnecessary but renamed the styles sheet to styles_bi (bi for busy indicator as it was conflicting when i was testing with other code). hope it is useful. happened to me when i had a listview in the page. basically i just set an explicit style for the state change that happenes when hide is invoked.

**Comment by RealOrangeOne** [comment]
the conflicting of styles doesnt sound right, if it were written with `var` then maybe. Could you try swapping it back to `style` and see if it's still an issue.

Also, could you remove the commented out styles if they arent being used / needed.

**Comment by temitope** [comment]
@RealOrangeOne pardon my unclear explanation, the style sheet name change is entirely unnecessary as a module, it is an artifact of me pulling in the component as a raw copy-paste to test the "no show" styling i added in. I will remove those additions ans resubmit.

File: index.js [hunk 1]
```diff
@@ -93,8 +99,8 @@ const BusyIndicator = React.createClass({
       }
     });
 
-    if (!this.state.isVisible) {
-      return (<View />);
+    if (!this.state.isVisible) {
```
Comment (RealOrangeOne) [inline index.js]: Looks like you have some trailing whitespace, could you remove

Comment (temitope) [reply index.js]: oops. done.

**Review by RealOrangeOne — APPROVED** [review]

**Pull request merged**